ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -247,13 +247,20 @@ ompi_mtl_ofi_send_start(struct mca_mtl_base_module_t *mtl,
     endpoint = ompi_mtl_ofi_get_endpoint(mtl, ompi_proc);
 
     ompi_ret = ompi_mtl_datatype_pack(convertor, &start, &length, &free_after);
-    if (OMPI_SUCCESS != ompi_ret) return ompi_ret;
+	if (OPAL_UNLIKELY(OMPI_SUCCESS != ompi_ret)) {
```

Comment:
We have a very small number of style guidelines (https://github.com/open-mpi/ompi/wiki/CodingStyle), but one of them is: absolutely no tabs whatsoever.  Always spaces.  😄

Reply:
Doh! okay.